tests/framework/artifacts.py

```diff
@@ -304,7 +304,7 @@ class ArtifactCollection:
     PLATFORM = platform.machine()
 
     # S3 bucket structure.
-    ARTIFACTS_ROOT = 'ci-artifacts'
+    ARTIFACTS_ROOT = 'ci-artifacts-dev'
```

Comment:
let's maybe also add the temp docker container tag in this commit, to make sure that we don't forget abt it

Reply:
I switched to using the production bucket.